jetstream/core/proto/jetstream.proto

```diff
@@ -23,6 +23,8 @@ service Orchestrator {
   rpc Decode(DecodeRequest) returns (stream DecodeResponse) {}
   // Checks if the model server is live.
   rpc HealthCheck(HealthCheckRequest) returns (HealthCheckResponse) {}
+  // Warms up the model server.
+  rpc ModelWarmup(ModelWarmupRequest) returns (ModelWarmupResponse) {}
```

Comment:
@FanhaiLu1 I was wondering would we need to expose a new API for switching warmup, in the long term, I was assuming that this will be enabled by default, and adding a new API field may be hard to deprecate later. But there is no other obvious downside as well

Reply:
I think given that there is the performance diff at the moment, we would need to expose a new API for enabling and disabling warmup. Would deprecation process be difficult in the future? I would assume that at the moment we wouldn't enable using model warmup by default. @FanhaiLu1

Reply:
It's depends. even in long term, this is option user can chose do warm up or not.

Reply:
Let's just put this into `service Orchestrator`

Reply:
Should we put this in `service Orchestrator` or its own service? I thought we want to decouple model warmup from orchestrator entirely? @FanhaiLu1

Reply:
Its own service is better.

Reply:
Thinking about this again, do you mean instead of having an endpoint, bake the AOT warmup logic into init based on if a user passes in a flag on model server startup say `--warmup = True`? This would remove the need of a model server warmup endpoint, but if a user decides they want to enable warmup, then they would have to restart deployment of the model server.

Reply:
--warmup = True with a flag looks good. Engineer who start the server know they want to warmup or not warmup.

Reply:
Sure, I think once we fix the OOM / performance gap we can switch to use flag.